__tests__/app.test.js

```diff
@@ -17,22 +17,69 @@ describe('/api/topics', () => {
         return request(app)
             .get('/api/topics')
             .expect(200)
-            .then((response) => {
-                expect(response.body.topics).toEqual(expect.any(Array));
-                expect(Object.keys(response.body.topics[0])).toEqual(
+            .then(({ body }) => {
+                const { topics } = body;
+                expect(topics).toEqual(expect.any(Array));
+                expect(Object.keys(topics[0])).toEqual(
                     expect.arrayContaining(['slug', 'description'])
                 );
-            })
+                topics.forEach((topic) => {
```

Comment:
Not sure why this test has been changed for this branch. If you are going to use a forEach in a test you must assert the length of the array as we can iterate over an empty array.